CHANGELOG.md

```diff
@@ -14,7 +14,15 @@ The following is a curated list of changes in the Enact agate module, newest cha
 - `agate/Picker` prop `noAnimation` and `wrap`
 - `agate/RangePicker` prop `noAnimation` and `wrap`
 - `agate/TimePicker` prop `onSpotlightDisappear` and `spotlightDisabled`
- 
+
+### Changed
+
+- `agate/ArcPicker` to read out audio guidance when focused via 5-way keys
+- `agate/ArcSlider` to read out audio guidance when focused via 5-way keys
+- `agate/FanSpeedControl` to read out audio guidance when focused via 5-way keys
+- `agate/TemperatureControl` to read out audio guidance when focused via 5-way keys
+- `agate/WindDirectionControl` to read out audio guidance when focused via 5-way keys
```

Comment:
Why did you repeat the same log? you could do like
`agate/ArcPicker`, `agate/ArcSlider`, ... in one log.